### ISSUE TITLE
Possibly wrong assertion in solid 1D section property card

### ISSUE THREAD

**Issue by JohnDN90** [MASTmultiphysics/mast-multiphysics#69]
@manavbhatia  [Here](https://github.com/MASTmultiphysics/mast-multiphysics/blob/master/src/property_cards/solid_1d_section_element_property_card.cpp#L1499), should  `libmesh_assert(!_initialized);` actually be `libmesh_assert(_initialized);`?  

The unique_ptr should already be pointing to null before initialization and logically you would only need to clear your property card, after it has been initialized, not before.  If I'm missing something, could you explain what it is?

**Comment by manavbhatia** [comment]
Did you mean to point to [clear()](https://github.com/MASTmultiphysics/mast-multiphysics/blob/47709572a1aef890f4ed70efae99bbd69824141e/src/property_cards/solid_1d_section_element_property_card.cpp#L1623) instead? 

I think we can remove this assert statement. It should not matter whether or not the object is initialized since all the `unique_ptr`s are cleared anyways.

**Comment by JohnDN90** [comment]
Yes, my link was actually pointing to clear() when this issue was created, but then the line numbers changed and the link was no longer correct after we merged in some changes last week.  Going forward, it might be better for me to reference the name of the file and the name of the function in the link text, so if line numbers change we don't run into this confusion.

And that sounds good.  This assert statement was causing one of my unit tests to fail when I tried clearing a section property card after I had initialized it.
